diff --git a/node_modules/notion-client/build/index.js b/node_modules/notion-client/build/index.js
index 2ad4eba..0377544 100644
--- a/node_modules/notion-client/build/index.js
+++ b/node_modules/notion-client/build/index.js
@@ -432,7 +432,7 @@ var NotionAPI = class {
   }
   async getBlocks(blockIds, kyOptions) {
     return this.fetch({
-      endpoint: "syncRecordValues",
+      endpoint: "syncRecordValuesMain",
       body: {
         requests: blockIds.map((blockId) => ({
           // TODO: when to use table 'space' vs 'block'?
