diff --git a/node_modules/notion-client/build/index.js b/node_modules/notion-client/build/index.js
index 2ad4eba..6419b9e 100644
--- a/node_modules/notion-client/build/index.js
+++ b/node_modules/notion-client/build/index.js
@@ -502,12 +502,13 @@ var NotionAPI = class {
       headers["x-notion-active-user-header"] = this._activeUser;
     }
     const url = `${this._apiBaseUrl}/${endpoint}`;
-    const res = await ky.post(url, {
-      mode: "no-cors",
-      ...this._kyOptions,
-      ...kyOptions,
-      json: body,
-      headers
+    const res = await fetch(url, {
+      method: 'POST',                
+      headers: {
+        'Content-Type': 'application/json',
+        ...headers,
+      },
+      body: JSON.stringify(body),    
     });
     return res.json();
   }
