diff --git a/node_modules/react-notion-x/build/index.js b/node_modules/react-notion-x/build/index.js
index 4403ab4..ce270a5 100644
--- a/node_modules/react-notion-x/build/index.js
+++ b/node_modules/react-notion-x/build/index.js
@@ -310,6 +310,7 @@ function LazyImage({
 }) {
   var _a, _b, _c;
   const { recordMap, zoom, previewImages, forceCustomImages, components } = useNotionContext();
+
   const zoomRef = React.useRef(zoom ? zoom.clone() : null);
   const previewImage = previewImages ? (_c = (_a = recordMap == null ? void 0 : recordMap.preview_images) == null ? void 0 : _a[src]) != null ? _c : (_b = recordMap == null ? void 0 : recordMap.preview_images) == null ? void 0 : _b[normalizeUrl(src)] : null;
   const onLoad = React.useCallback(
@@ -684,8 +685,10 @@ function Asset({
   if (!source) {
     return null;
   }
+
   if (block.space_id) {
     const url = new URL(source);
+
     url.searchParams.set("spaceId", block.space_id);
     source = url.toString();
   }
@@ -784,9 +787,12 @@ function Asset({
       }
     }
   } else if (block.type === "image") {
-    if (source.includes("file.notion.so")) {
-      source = (_k = (_j = (_i = block.properties) == null ? void 0 : _i.source) == null ? void 0 : _j[0]) == null ? void 0 : _k[0];
+    if (!source.includes('.gif')) {
+      if (source.includes("file.notion.so")) {
+        source = (_k = (_j = (_i = block.properties) == null ? void 0 : _i.source) == null ? void 0 : _j[0]) == null ? void 0 : _k[0];
+      } 
     }
+
     const src = mapImageUrl(source, block);
     const caption = getTextContent((_l = block.properties) == null ? void 0 : _l.caption);
     const alt = caption || "notion image";
@@ -1447,6 +1453,7 @@ var wrapNextImage = (NextImage) => {
     if (fill === "undefined") {
       fill = !(width && height);
     }
+
     return /* @__PURE__ */ jsx16(
       NextImage,
       {
@@ -1475,6 +1482,7 @@ var wrapNextLegacyImage = (NextLegacyImage) => {
     if (!layout) {
       layout = width && height ? "intrinsic" : "fill";
     }
+
     return /* @__PURE__ */ jsx16(
       NextLegacyImage,
       {
